src/configurations/cancercomplexity/routesConfig.ts

```diff
@@ -199,6 +199,18 @@ const routes: GenericRoute[] = [
                         ...projectCardConfiguration,
                       },
                     },
+                    {
+                      name: 'CardContainerLogic',
+                      columnName: 'grantNumber',
+                      title: 'Related People',
+                      tableSqlKeys: ['grantNumber'],
+                      props: {
+                        sqlOperator: 'LIKE',
```

Comment:
Since grantNumber is a StringList in the [people table](https://www.synapse.org/#!Synapse:syn28073190/tables/), does the sqlOperator 'HAS' work here?

Reply:
You're right!

I'm curious - is there a performance difference between using `LIKE` and `HAS`?

Reply:
@marcomarasca can probably provide more info.  My understanding is that it should have improved database performance (HAS should be looking for the exact string match), but probably not noticeable for this amount of data.

Reply:
@vpchung : If HAS works here, could you commit that change (and I will approve/merge the PR)?

Reply:
Done! Sorry for the wait.